src/Cli/dotnet/commands/dotnet-workload/install/LocalizableStrings.resx

```diff
@@ -169,7 +169,7 @@
     <value>Download of workload packs to offline cache failed: {0}</value>
   </data>
   <data name="WorkloadAlreadyInstalled" xml:space="preserve">
-    <value>Workload '{0}' is already installed.</value>
+    <value>Workload(s) '{0}' are already installed. Run `dotnet workload update` to update your workloads.</value>
```

Comment:
By default if you run `dotnet workload install workload1 workload2`, it will already update the workloads.  You would have to specify `--skip-manifest-update` to avoid the update.  So does it make sense to include the second sentence in this message here?

Reply:
Good point, made more sense when this was an error. I'll remove.